composer/datasets/ade20k_hparams.py

```diff
@@ -180,6 +180,7 @@ class StreamingADE20kHparams(DatasetHparams):
 
     def initialize_object(self, batch_size: int, dataloader_hparams: DataLoaderHparams) -> DataSpec:
         if self.version == 1:
+            warn_streaming_dataset_deprecation(old_version=1, new_version=2)
```

Comment:
Used the variable "version" here. I will change it in next commit.

Reply:
Done in https://github.com/mosaicml/composer/pull/1532/commits/15c074fbf22d41d2f7ad0e87043dae36eec88e0f.